web-client/src/presenter/actions/CreateAccount/createNewPetitionerUserAction.tsx

```diff
@@ -0,0 +1,66 @@
+import { NewPetitionerUser } from '@shared/business/entities/NewPetitionerUser';
+import { state } from '@web-client/presenter/app.cerebral';
+import React from 'react';
+
+export const createNewPetitionerUserAction = async ({
+  applicationContext,
+  get,
+  path,
+}: ActionProps) => {
+  const petitionerAccountForm = new NewPetitionerUser(
+    get(state.form),
+  ).toRawObject();
+
+  try {
+    const response = await applicationContext
+      .getUseCases()
+      .signUpUserInteractor(applicationContext, {
+        user: petitionerAccountForm,
+      });
+
+    return path.success(response);
+  } catch (err: any) {
+    const originalErrorMessage = err?.originalError?.response?.data;
+
+    if (originalErrorMessage === 'User already exists') {
+      return path.warning({
+        alertWarning: {
+          message: (
+            <>
+              This email address is already associated with an account. You can{' '}
+              <a href="/login">log in here</a>. If you forgot your password, you
+              can <a href={'/forgot-password'}> request a password reset</a>.
+            </>
```

Comment:
intriguing usage of jsx in the actions. i don't think we've seen that in the project yet. makes sense to me even if I wouldn't have thought to do it this way.